help/markdown/fake-tools-pickles.md

```diff
@@ -22,7 +24,8 @@ Target.create "BuildDoc" (fun _ ->
     Pickles.convert (fun p ->
         { p with FeatureDirectory = currentDirectory </> "Specs"
                  OutputDirectory = currentDirectory </> "SpecDocs"
-                 OutputFileFormat = Pickles.DocumentationFormat.DHTML })
+                 OutputFileFormat = Pickles.DocumentationFormat.DHTML
+                 ToolPath = Tools.findToolInSubPath "pickles.exe" "~/.fake/build.fsx/packages" })
```

Comment:
This is not something we should suggest. I consider the `.fake` directory basically implementation detail. I can see the problem but I don't have a good solution. Maybe we can automatically search in the users `.nuget` directory. This way it doesn't need to be copied in `.fake`.

Reply:
@matthid Fine. Is there already a helper to get the .nuget directory of the user? 
Because nearly all of my PR's (ReportGenerator, Pickles and SpecFlow) depend on a external NuGet package with the tool or the tool itself already installed. 
I haven't found a different way yet. Maybe by setting the path in the Paket storage option explicitely?

Reply:
@matthid Ok, I have a look then